yt_dlp/cookies.py

```diff
@@ -146,7 +146,8 @@ def _extract_firefox_cookies(profile, container, logger):
             identities = json.load(containers).get('identities', [])
         container_id = next((context.get('userContextId') for context in identities if container in (
             context.get('name'),
-            try_call(lambda: re.fullmatch(r'userContext([^\.]+)\.label', context['l10nID']).group())
+            try_call(lambda: re.fullmatch(r'userContext([^.]+)\.label', context['l10nID']).group()),
+            try_call(lambda: re.fullmatch(r'user-context-(.+)', context['l10nId']).group())
```

Comment:
another note: this code has always been wrong? shouldn't it be `group(1)` instead of `group()`?

Reply:
possible solution is to do this, but will `title()` work with all the preset container names?
```suggestion
            try_call(lambda: re.fullmatch(r'userContext([^.]+)\.label', context['l10nID']).group(1)),
            try_call(lambda: re.fullmatch(r'user-context-(.+)', context['l10nId']).group(1).title())
```